docs/_snippets/framework/tutorials/external-widget.js

```diff
@@ -73,9 +83,9 @@ class ExternalWidgetEditing extends Plugin {
 	constructor( editor ) {
 		super( editor );
 
-		this._updateWidgetDataFn = this._updateWidgetData;
-
 		this.intervalId = this._intervalFetch();
+
+		this.externalDataValue = 'Fetching data...';
```

Comment:
I would initialize this with an empty string. This value is a placeholder, not a real value.